c7n/cli.py

```diff
@@ -401,3 +401,7 @@ def main():
             raise
         traceback.print_exc()
         pdb.post_mortem(sys.exc_info()[-1])
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
i typically just do `python setup.py develop` so that the checkout is effectively live in the virtualenv, ie instead of setup.py install which does a copy.